.github/workflows/ci.yml

```diff
@@ -14,33 +14,33 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ '7.4', '8.2' ]
+                php-version: [ '8.1', '8.4' ]
```

Comment:
I think this just tests 8.1 and 8.4.  Don't you need to list all versions?

Reply:
I think it has always been min and max version only. It's just to make sure that min version still works.